### PR TITLE
Trying to implement Issue #75

### PR DESCRIPTION
**Pull request by william1657**
One last try.

### CODE REVIEW

**Review by oliver-ni — CHANGES_REQUESTED** [review]
Thanks—this is really nice. One nitpick: can we actually get this merged into the spawn message? Right now it's sending two messages right next to each other

**Comment by william1657** [comment]
This should merge it into the spawn message.

File: cogs/spawning.py [hunk 1]
```diff
@@ -236,6 +236,11 @@ async def on_message(self, message: discord.Message):
             self.spawn_threshold *= 1.1
 
     async def spawn_pokemon(self, channel, species=None, incense=None):
+        prev_species = ""
```
Comment (oliver-ni) [inline cogs/spawning.py:239]: Last nitpick, I think it'd be better to have a `None` here instead of empty string. Otherwise, looks good!

**Review by oliver-ni — CHANGES_REQUESTED** [review]

File: cogs/spawning.py (hunk 1 above)
Comment (william1657) [reply cogs/spawning.py]: Changed

**Comment by oliver-ni** [comment]
Thank you!

**Pull request merged**